app/v2/notifications/post_notifications.py

```diff
@@ -149,7 +151,8 @@ def post_notification(notification_type):
                 api_key=api_user,
                 template=template,
                 service=authenticated_service,
-                reply_to_text=reply_to
```

Comment:
Since you're in here, may as well fix the formatting. Add spaces around the equal for readability.